Dockerfile

```diff
@@ -1,7 +1,14 @@
-FROM jinaai/jina:master
+FROM jinaai/jina:2.0.0rc6
```

Comment:
GPU support needs to be added. You can pair with @bwanglzu. He implemented the same with text.

Reply:
How are we testing this?